src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.scss

```diff
@@ -83,3 +92,19 @@
 .btn-circle:active {
     background-color: var(--iris-btn-circle-hover);
 }
+
+@keyframes blink {
+    0% {
+        opacity: 1;
+    }
+    50% {
```

Comment:
Codacy

Reply:
sorry but I don't understand what you mean

Reply:
Codacy complains about not having an empty line here. You can look at the style issues either in the GitHub Checks or in the changed files view of your PR

Reply:
ahh thank you very much! @JohannesStoehr